FurCoNZ/Startup.cs

```diff
@@ -234,6 +244,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                     name: "default",
                     template: "{controller=Home}/{action=Index}/{id?}");
             });
+
+            // Aparently stripe uses a singleton pattern 😟
+            Stripe.StripeConfiguration.SetApiKey(Configuration.GetValue<string>("Stripe::SecretKey"));

```

Comment:
I think you're only needing a single colon here... <insert [Unexpected T_PAAMAYIM_NEKUDOTAYIM](https://phil.tech/php/2013/09/09/t-paamayim-nekudotayim-v-sanity/) joke here>
```suggestion
            Stripe.StripeConfiguration.SetApiKey(Configuration.GetValue<string>("Stripe:SecretKey"));
```